public/lib/calendar_vis_config.js

```diff
@@ -67,8 +68,8 @@ export class CalendarVisConfig extends BaseConfig {
       min: from,
       max: to
     } = timeFilter.getBounds();
-    const diff = to.diff(from, 'days');
-    if (diff <= 1) {
+
+    if ( moment(new Date(from)).format("MMM Do YY") === moment(new Date(to)).format("MMM Do YY") ) {
```

Comment:
remove the blank space beginning from the brace, and use `MMM Do YYYY` to format would be better to avoid cases like different years (e.g. 1919, 2019)